src/lib.rs

```diff
@@ -549,30 +582,31 @@ impl<'a> Server<'a> {
 	///
 	/// [`Plugin::plugin_message`]: plugin/trait.Plugin.html#method.plugin_message
 	pub fn send_plugin_message<S: AsRef<str>>(&self, message: S) {
-		unsafe {
-			let text = to_cstring!(message.as_ref());
-			(TS3_FUNCTIONS.as_ref().expect("Functions should be loaded").send_plugin_command)(
-				self.get_id().0,
-				to_cstring!(self.api.get_plugin_id()).as_ptr(),
-				text.as_ptr(),
-				PluginTargetMode::Server as i32,
-				std::ptr::null(),
-				std::ptr::null(),
-			);
-		}
+		let text = to_cstring!(message.as_ref());
+		(TS3_FUNCTIONS
+			.lock()
+			.unwrap()
+			.as_ref()
+			.expect("Functions should be loaded")
+			.send_plugin_command)(
+			self.get_id().0,
+			to_cstring!(self.api.get_plugin_path()).as_ptr(),
```

Comment:
Passing the plugin path here seems weird, it should be the plugin id like before.